stubs/D3DShot/d3dshot/capture_output.pyi

```diff
@@ -0,0 +1,36 @@
+import enum
+from _typeshed import Incomplete
+from ctypes import _CVoidConstPLike
+from typing_extensions import Literal, TypeAlias
+
+from PIL import Image
+
+_Frame: TypeAlias = Image.Image | Incomplete
+# _Frame: TypeAlias = Image.Image | npt.NDArray[np.int32] | npt.NDArray[np.float32] | _Tensor
```

Comment:
Add a reference to #5768 so it shows up when #5768 is fixed and we start grepping for all the references in the codebase to that issue

```suggestion
# Fix when #5768 is resolved:
# _Frame: TypeAlias = Image.Image | npt.NDArray[np.int32] | npt.NDArray[np.float32] | _Tensor
```